source/clear-linux/get-started/virtual-machine-install/hyper-v.rst

```diff
@@ -12,26 +12,19 @@ Please ensure you have enabled `Intel® Virtualization Technology
 <https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices>`_
 (Intel® VT-d) in your BIOS/UEFI firmware configuration.
 
-Enable Hyper-V
-==============
+Install Hyper-V
```

Comment:
It actually is enabling hyper-v, not installing it, which is the reason I changed it last update.  Hyper-v comes installed on Win10 pro and it has to be enabled